app/src/main/java/com/example/gfood/utils/ViewExt.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.gfood.utils
+
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+fun ImageView.loadImage(
+    url: String)
+{
```

Comment:
sao function viết ẩu thế ? refactor lại code trước khi commit chứ 
https://kotlinlang.org/docs/coding-conventions.html#class-headers
```suggestion
fun ImageView.loadImage(url: String) {
```